src/RecoveryServices/RecoveryServices/Vault/NewAzureRMRecoveryServicesVault.cs

```diff
@@ -74,6 +74,24 @@ public class NewAzureRmRecoveryServicesVault : RecoveryServicesCmdletBase
         [Parameter(Mandatory = false)]
         public bool? DisableAzureMonitorAlertsForJobFailure { get; set; }
 
+        /// <summary>
+        /// Enables or disables classic email notifications for Site Recovery in RS vault.
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        public bool? DisableEmailNotificationsForSiteRecovery { get; set; }
+
+        /// <summary>
+        /// Enables or disables monitor alerts for replication issue in RS vault.
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        public bool? DisableAzureMonitorAlertsForAllReplicationIssue { get; set; }
```

Comment:
Hi @Nickcandy 

I have updated it from "DisableAzureMonitorAlertsForAllReplicationIssues" to "DisableAzureMonitorAlertsForAllReplicationIssue" as static analysis failed. Is it not possible to get exception for this parameter to be DisableAzureMonitorAlertsForAllReplicationIssues as it looks correct and required as per PM ?